src/praisonai-agents/praisonaiagents/tools/messaging_tools.py

```
@@ -139,6 +139,54 @@ def _parse_media(message: str) -> tuple[str, Optional[List[str]]]:
+        return None
+
+    agent_id = session_id = ""
+    origin = None
+    try:
+        ctx = get_session_context()
+        agent_id = getattr(ctx, "user_id", "") or ""
+        session_id = getattr(ctx, "chat_id", "") or ""
+        if getattr(ctx, "origin", None) is not None and ctx.origin.platform:
+            origin = "origin"
+    except Exception:
+        pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)